test/extended/cmd.sh

```diff
@@ -143,3 +144,15 @@ os::cmd::expect_success "oc process -f test/extended/fixtures/image-pull-secrets
 os::cmd::try_until_text "oc get pods/my-dc-1-hook-pre -o jsonpath='{.status.containerStatuses[0].imageID}'" "docker"
 os::cmd::expect_success "oc delete all --all"
 os::cmd::expect_success "docker rmi -f ${docker_registry}/image-ns/busybox:latest"
+
+# check to make sure that service serving cert signing works correctly
```

Comment:
I don't want to do a build in the tests to test something that doesn't need
builds fundamentally (because they are much slower and have a bigger system
impact).  I'll relax the extended test but please fix this.

On Fri, Apr 29, 2016 at 7:37 AM, David Eads <notifications@github.com>
wrote:

> In test/extended/cmd.sh
> <https://github.com/openshift/origin/pull/7728#discussion_r61564349>:
>
> > @@ -143,3 +144,15 @@ os::cmd::expect_success "oc process -f test/extended/fixtures/image-pull-secrets
> >  os::cmd::try_until_text "oc get pods/my-dc-1-hook-pre -o jsonpath='{.status.containerStatuses[0].imageID}'" "docker"
> >  os::cmd::expect_success "oc delete all --all"
> >  os::cmd::expect_success "docker rmi -f ${docker_registry}/image-ns/busybox:latest"
> > +
> > +# check to make sure that service serving cert signing works correctly
>
> Does it need to do a build, or was that just convenient?
>
> It was convenient. I didn't look into what format nginx expected. Since
> this is about how its test, not whether its tested can it be a follow on?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/7728/files/e3f4db09aa1f5cc71b10ecb2b076070ceedfde3f#r61564349>
>